beta-src/src/models/NodeSetClass.ts

```diff
@@ -0,0 +1,180 @@
+import PathSearchClass from "./PathSearchClass";
+import TerritoryClass from "./TerritoryClass";
+
+export default class NodeSetClass {
+  Nodes: Record<string, TerritoryClass> = {};
+
+  Path: any;
+
+  addNodes(Nodes: TerritoryClass[]) {
+    Nodes.map((n) => {
+      return this.addNode(n);
+    });
+  }
+
+  addNode(Node: TerritoryClass) {
+    this.Nodes[Node.id] = Node;
+  }
+
+  initNodes(search) {
+    Object.values(this.Nodes).map((n) => {
+      return n.nodeInit(search);
+    });
+  }
+
+  routeSetLoad(ConvoyGroup) {
+    ConvoyGroup.fleets.map((f) => {
+      return this.addNode(f.Territory);
+    });
+
+    ConvoyGroup.armies.map((a) => {
+      return this.addNode(a.Territory);
+    });
+
+    ConvoyGroup.coasts.map((c) => {
+      return this.addNode(c);
+    });
+  }
+
+  routeSetStart(StartTerr, fEndNode, fAllNode, fAnyNode) {
```

Comment:
Where do `StartTerr` and `EndTerr` come from? Can we maintain consistency by making these camel case? Only constants or enums should have a different case. So `StartTerr` should be `startTerr`, etc. Also, please change `cur` to `curr` for more clarity and consistency.

Reply:
as per conversation we had, I've just ported in convoy path related codes. I did not make any changes to the code except removing Prototype methods.